packages/quantic/playwright/page-object/caseAssistObject.ts

```diff
@@ -0,0 +1,27 @@
+import {Page, Locator, Response} from '@playwright/test';
+import {
+  classifyRequestRegex,
+  documentsSuggestRequestRegex,
+} from '../utils/requests';
+
+export class CaseAssistObject {
+  constructor(private page: Page) {
+    this.page = page;
+  }
+
+  get fetchClassificationsButton(): Locator {
+    return this.page.locator('c-action-fetch-classifications button');
+  }
+
+  async fetchClassifications(): Promise<void> {
+    await this.fetchClassificationsButton.click();
+  }
+
+  async waitForCaseClassificationsResponse(): Promise<Response> {
+    return this.page.waitForResponse(classifyRequestRegex);
+  }
+
+  async waitForDocumentSuggestionResponse(): Promise<Response> {
+    return this.page.waitForResponse(documentsSuggestRequestRegex);
+  }
```

Comment:
we have nothing to check in the body of the requests here? I believe the cypress tests were checking for example the value and the searchhub